src/registrar.js

```diff
@@ -233,6 +242,9 @@ export default class Registrar {
           ret.gracePeriodEndDate = gracePeriodEndDate
         }
       }
+      if (legacyEntry.registrationDate && permEntry.registrationDate) {
```

Comment:
`
An auction for this name was started in 2018, but the auction never completed. The name was first registered recently, but the manager shows a registration date in 2018.
`
Doesn't this pick `permEntry.registrationDate`, regardless of auction completed or not?

For example, apt-get.eth was one of the names registered during the auction but showing the last registered name using your PR branch (on the left).
<img width="1426" alt="Screenshot 2021-12-15 at 12 34 58" src="https://user-images.githubusercontent.com/2630/146118534-f2a343bd-6926-43cd-905c-8897cb6990e0.png">

Reply:
Thank you! Yes, you are right. After having a closer look, I wonder if the ens manager app should even display the registration date when the auction never completed in the legacy registrar to remain consistent with current behavior of permanent registrar (i.e. registration date is not reported for names registered with permanent registrar). That could be a new feature though. Last commit resets the registration date to 0 when the auction never finalized so that the registration date does not show up. Finally, I am not sure if this validity check should be implemented in the ensdomains/ui library or in the ens manager app (`getRegistrarEntry()` in resolvers.js).